doc/api/crypto.md

```diff
@@ -1380,6 +1386,311 @@ Based on the recommendations of [NIST SP 800-131A][]:
+<td>Instructs OpenSSL to disable a SSL 3.0/TLS 1.0 vulnerability workaround added in OpenSSL 0.9.6d.</td>
+</tr>
+<tr>
+<td><code>SSL_OP_EPHEMERAL_RSA</code></td>
+<td>Instructs OpenSSL to always use the tmp_rsa key when performing RSA operations.</td>
+</tr>
+<tr>
+<td><code>SSL_OP_LEGACY_SERVER_CONNECT</code></td>
+<td>Allow initial connection to servers that do not support RI.</td>
+</tr>
+<tr>
+<td><code>SSL_OP_MICROSOFT_BIG_SSLV3_BUFFER</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_MICROSOFT_SESS_ID_BUG</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_MSIE_SSLV2_RSA_PADDING</code></td>
+<td>Instructs OpenSSL to disable the workaround for a man-in-the-middle
+protocol-version vulnerability in the SSL 2.0 server implementation.</td>
+</tr>
+<tr>
+<td><code>SSL_OP_NETSCAPE_CA_DN_BUG</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_NETSCAPE_CHALLENGE_BUG</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_NETSCAPE_DEMO_CIPHER_CHANGE_BUG</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_NETSCAPE_REUSE_CIPHER_CHANGE_BUG</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_NO_COMPRESSION</code></td>
+<td>Instructs OpenSSL to disable support for SSL/TLS compression.</td>
+</tr>
+<tr>
+<td><code>SSL_OP_NO_QUERY_MTU</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_NO_SESSION_RESUMPTION_ON_RENEGOTIATION</code></td>
+<td>Instructs OpenSSL to always start a new session when performing renegotiation.</td>
+</tr>
+<tr>
+<td><code>SSL_OP_NO_SSLv2</code></td>
+<td>Instructs OpenSSL to turn off SSL v2</td>
+</tr>
+<tr>
+<td><code>SSL_OP_NO_SSLv3</code></td>
+<td>Instructs OpenSSL to turn off SSL v3</td>
+</tr>
+<tr>
+<td><code>SSL_OP_NO_TICKET</code></td>
+<td>Instructs OpenSSL to disable use of RFC4507bis tickets.</td>
+</tr>
+<tr>
+<td><code>SSL_OP_NO_TLSv1</code></td>
+<td>Instructs OpenSSL to turn off TLS v1</td>
+</tr>
+<tr>
+<td><code>SSL_OP_NO_TLSv1_1</code></td>
+<td>Instructs OpenSSL to turn off TLS v1.1</td>
+</tr>
+<tr>
+<td><code>SSL_OP_NO_TLSv1_2</code></td>
+<td>Instructs OpenSSL to turn off TLS v1.2</td>
+</tr>
+<td><code>SSL_OP_PKCS1_CHECK_1</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_PKCS1_CHECK_2</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_SINGLE_DH_USE</code></td>
+<td>Instructs OpenSSL to always create a new key when using temporary/ephemeral
+DH parameters.</td>
+</tr>
+<tr>
+<td><code>SSL_OP_SINGLE_ECDH_USE</code></td>
+<td>Instructs OpenSSL to always create a new key when using temporary/ephemeral
+ECDH parameters.</td>
+</tr>
+<td><code>SSL_OP_SSLEAY_080_CLIENT_DH_BUG</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_SSLREF2_REUSE_CERT_TYPE_BUG</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_TLS_BLOCK_PADDING_BUG</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_TLS_D5_BUG</code></td>
+<td></td>
+</tr>
+<tr>
+<td><code>SSL_OP_TLS_ROLLBACK_BUG</code></td>
+<td>Instructs OpenSSL to disable version rollback attack detection.</td>
+</tr>
+</table>
```

Comment:
It's used in `streams.md`. Only difference is that that one is indented correctly.